Chat/Network.cpp

```diff
@@ -145,6 +145,31 @@ void Network::SendFile(std::string pass, std::string ip)
 	 file_send_socket_.SendFile(pass, ip);
 }
 
+std::vector<std::string> Network::GetFileList()
```

Comment:
It is good practice to pass output variable as function parameter. I mean GetFileList(std::vector<std::string>& ).